src/app/shared/layout/header.component.ts

```diff
@@ -31,7 +31,7 @@ export class HeaderComponent implements OnInit {
               public auth: AuthService,
               public phaseService: PhaseService,
               public userService: UserService,
-              private loggingService: LoggingService,
+              public loggingService: LoggingService,
```

Comment:
Hmm can consider keeping the `loggingService` private and wrapping `this.loggingService.exportLogFile()` in a public function under HeaderComponent.
I feel that this might be better style.

Reply:
@dingyuchen 
Thanks for the suggestion!
I agree your suggestion might be better since we only use one function of `loggingService`.
I'll make the changes.

Reply:
@dingyuchen can help to merge after these changes? Thanks! :+1: